5_models/Modeling_Lesson.Rmd

```diff
@@ -0,0 +1,224 @@
+---
+title: "R Tutorial: Modeling"
+output: 
+  html_document: 
+    toc: true
+    toc_float: true
+--- 
+
+ Welcome to the lesson on modeling in R! 
+Goals: 
+1. Learn about functions and syntax for models in R 
+2. Practice examples of different model types 
+3. Understand how to obtain model results in R  
+4. Learn how to visualize the results of the models
+
+ Let's get some data. 
+We will use the same data we are already familiar with from the Descriptives lesson. We will compute a group for high and low ADHD, so we can use that in our models. 
+```{r}
+
+df <- read.csv("uncapher_2016_repeated_measures_dataset.csv")
+head(df)
+```
+
+clean up the data for these models
+```{r}
+library(tidyverse)
+ df <- df %>%   
+    # only keep data from the task when distractor was present
+   dplyr::filter(numDist == 6) %>%
+   # only include high or low groups for multitasking.
+   dplyr::filter(groupStatus != "") %>%
+   # if adhd score is lower than mean, label "low", else label "high""
+   mutate(adhdF = as.factor(ifelse(adhd < mean(adhd), "Low ADHD", "High ADHD")))
+  
+names(df)
+```
+
+# Basics for modeling in R 
+
+## Functions:
+### Basic models 
+For each model type, there is a different function. Many of these are available in base R.
+
+TTEST: t.test()
+CORRELATION: cor.test()
+LINEAR REGRESSION: lm()
+ANOVA: aov()
+
+Each function has their own set of arguments, based on the options available. 
+Always use the ? to see more details for the model you're using.
+
+The first argument is always the model formula, which follows a standard syntax.
+
+## Syntax: 
+### Basic model formula
+For any type of model with an IV and DV, there is a common syntax. The "~" delineates the direction of the model, where the DV is always on the left of the ~ and the predictors are always on the right.
+
+Y ~ X
+
+Example of t-test
+```{r}
+# ttest looking at the effect of ADHD group (IV) on dprime (DV)
+t.test(df$dprime ~ df$adhdF)
```

Comment:
I think we might want to teach only the second method you have here (using the data argument) as it's a little cleaner and encourages people to only t-test columns in the same df. We could probably just remove this part and only include the instructions for the second one

Reply:
agreed with monica! I'll update this